.github/workflows/cypress.yaml

```diff
@@ -0,0 +1,54 @@
+# Run the tests
+# Todo : reuse a build from another action, instead of rebuilding for nothing.
+# tchap modification : use "./scripts/layered.with.version.sh" instead of "./scripts/layered.sh"
+name: Cypress End to End Tests
+on:
+  pull_request: { }
+  push:
+    branches: [ develop, master ]
```

Comment:
est ce qu'il faudrait rajouter `develop_tchap`?

Reply:
Removed developed and master, which are element branches, and added develop_tchap and master_tchap, which we are using. (master_tchap actually doesn't exist yet since we haven't released yet)

We could even block commits to master and develop, to be sure things are clear...

Reply:
Yes we could but that would be in a different workflow I guess